docs/config/README.md

```diff
@@ -1753,6 +1754,39 @@ By default the module will be shown if any of the following conditions are met:
 symbol = "∴ "
 ```
 
+## localip
+
+The `localip` module shows the IPv4 address of the primary network interface.
+
+### Options
+
+| Option     | Default                     | Description                                                                                                                          |
+| ---------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| `ssh_only` | `true`                      | Only show IP address when connected to an SSH session.                                                                               |
+| `format`   | `"[$hostname]($style)"`     | The format for the module.                                                                                                           |
+| `style`    | `"bold yellow"`              | The style for the module.                                                                                                            |
+| `disabled` | `false`                     | Disables the `localip` module.                                                                                                      |
+
+### Variables
+
+| Variable | Example | Description                          |
+| -------- | ------- | ------------------------------------ |
+| symbol   |         | Mirrors the value of option `symbol` |
```

Comment:
This also seems wrong, since there is no `symbol` option in this module.

In your defense, it seems you copied this from the hostname module, and I think that module is just wrong as well....gonna go open a PR on it once I finish here.